Common/Vulkan/VulkanImage.cpp

```diff
@@ -76,6 +77,8 @@ bool VulkanTexture::CreateDirect(VkCommandBuffer cmd, int w, int h, int numMips,
 	if (allocator_) {
 		offset_ = allocator_->Allocate(mem_reqs, &mem_, Tag());
 		if (offset_ == VulkanDeviceAllocator::ALLOCATE_FAILED) {
+			vkDestroyImage(vulkan_->GetDevice(), image_, nullptr);
```

Comment:
Destroy is called in the destructor which already does this.  We should at least clear `image_`, but I'm personally not a big fan of repeating WET cleanup code in every error case, so I try to design things so they auto-cleanup and the code is easier to read.

-[Unknown]

Reply:
WET?  Yeah I see your point, I'll clean it up.

Reply:
We Enjoy Typing.  The opposite of DRY.

-[Unknown]

Reply:
Ah :)